File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        N = len(arr)
        for x in range(N):
            if x+2 < N and arr[x] % 2 == 1  and arr[x+1] % 2 == 1 and arr[x+2] % 2 == 1:
                return True
        return False

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int n = arr.length;
        for(int i=0; i<n; i++){
            if(i+2<n && arr[i]%2==1 && arr[i+1]%2==1 && arr[i+2]%2==1) return true;
        }
        return false;
    }
}